providers/bitbucket_test.go

```diff
@@ -119,8 +120,9 @@ func TestBitbucketProviderGetEmailAddress(t *testing.T) {
 	bURL, _ := url.Parse(b.URL)
 	p := testBitbucketProvider(bURL.Host, "", "")
 
+	ctx := context.TODO()
```

Comment:
Please change the others in this file too
```suggestion
	ctx := context.Background()
```